lib/active_admin/application.rb

```diff
@@ -178,9 +178,9 @@ def namespace(name)
       name ||= :root
 
       namespace = namespaces[name] ||= begin
-        namespace = Namespace.new(self, name)
-        ActiveSupport::Notifications.publish ActiveAdmin::Namespace::RegisterEvent, namespace
-        namespace
```

Comment:
Commit removed. Apparently my request for better feedback on code reviews didn't succeed... :cry: